README.md

```diff
@@ -124,6 +124,9 @@ To uninstall the chart:
 | space.additionalLabels | Additional labels for Space Custom Resource | `{}` |
 | space.annotations | Annotations for Space Custom Resource | `{}` |
 | space.tenant | Tenant associated with Space Custom Resource | `""` |
+| namespace.enabled | Enable Space Custom Resource | `false` |
+| namespace.additionalLabels | Additional labels for Space Custom Resource | `{}` |
+| namespace.annotations | Annotations for Space Custom Resource | `{}` |
```

Comment:
all wrong descriptions! it's namespace not space

Reply:
fixed